docs/assert/timeout.md

```diff
@@ -44,3 +44,12 @@ QUnit.test( "Waiting for async function", function( assert ) {
   } );
 });
 ```
+
+```js
+QUnit.test( "Waiting in an async test", async assert => {
+  assert.timeout( 500 ); // Timeout of .5 seconds
+
+  var result = await asyncFunction();
```

Comment:
Should we use `let`/`const` instead? I don't particularly care, just seems a bit odd to be using `async`/`await` with `var`.

Reply:
Thanks, fixed.